brainscore/submission/configuration.py

```diff
@@ -60,4 +61,6 @@ def object_decoder(config, work_dir, config_path, db_secret, jenkins_id):
         return SubmissionConfig(model_type=config['model_type'], user_id=config['user_id'], work_dir=work_dir,
                                 config_path=config_path,
                                 jenkins_id=jenkins_id, db_secret=db_secret,
-                                public=config['public'] == "True")
+                                public=config['public'] == "True",
+                                competition_submission=config['competition'] == "cosyne2022")
```

Comment:
I thought we were going to write the competition name, i.e. `"cosyne2022"` and not a bool?